plutus-tx-plugin/test/IsData/9.2/unsafeDeconstructData.pir-readable.golden

```diff
@@ -1,30 +1,29 @@
 let
   data Unit | Unit_match where
     Unit : Unit
+  !error : all a. unit -> a = /\a -> \(thunk : unit) -> error {a}
+  ~reconstructCaseError : string = "PT1"
+  !trace : all a. string -> a -> a = trace
+  !unitval : unit = ()
+  ~fallthrough : all a. a
```

Comment:
This is probably also a reason we got saved sometimes: if `fallthrough` is polymorphic then the type abstraction saves us.